packages/ui/__tests__/Link.test.tsx

```diff
@@ -52,29 +54,33 @@ describe('Link', () => {
   })
 
   it('Renders the Tooltip with correct props', async () => {
-    const wrapper = await mountAndCheckA11Y(
-      <Link type="tooltip" tooltip="Check our website" href="https://hazelcast.com/">
-        Tooltip Link
-      </Link>,
-    )
+    // This has to be wrapper in act() - you can find an explanation in Tooltip test file.
```

Comment:
Created a PR with a proposal on how to fix it https://github.com/hazelcast/frontend-shared/pull/47

Reply:
Smart, thanks for this PR! :)